### ISSUE TITLE
Chef fail to restart apache when php_version is specified in site.yml

### ISSUE THREAD

**Issue by katanyan** [vccw-team/vccw#84]
I tried to 'vagrant up' with php_version,  and then I got chef abend with apache restarting probrem.
Commands are as below.

```
$ unzip vccw-2.13.0.zip
$ cd vccw-2.13.0
$ cp provision/default.yml site.yml
$ vi site.yml
----
# set php_version to any version number
php_version: 5.3.3
----
$ vagrant up
```

Provision has been begun, but chef failed to restart apache. 

```
Bringing machine 'vccw.dev' up with 'virtualbox' provider...
==> vccw.dev: Importing base box 'miya0001/vccw'...
==> vccw.dev: Matching MAC address for NAT networking...

[...]

==> vccw.dev: [2015-08-10T03:38:48+00:00] INFO: template[/etc/httpd/sites-available/wordpress.conf] sending reload action to service[apache2] (delayed)
==> vccw.dev: Recipe: apache2::default
==> vccw.dev:
==> vccw.dev: * service[apache2] action reload
==> vccw.dev:  (up to date)
==> vccw.dev: [2015-08-10T03:38:48+00:00] INFO: bash[create-ssl-keys] sending restart action to service[apache2] (delayed)
==> vccw.dev:
==> vccw.dev: * service[apache2] action restart
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev: ================================================================================
==> vccw.dev:
==> vccw.dev: Error executing action `restart` on resource 'service[apache2]'
==> vccw.dev:
==> vccw.dev: ================================================================================
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev: Mixlib::ShellOut::ShellCommandFailed
==> vccw.dev:     ------------------------------------
==> vccw.dev:     Expected process to exit with [0], but received '1'
==> vccw.dev:     ---- Begin output of /sbin/service httpd restart && sleep 1 ----
==> vccw.dev:     STDOUT: Stopping httpd: [FAILED]
==> vccw.dev:     Starting httpd: [FAILED]
==> vccw.dev:     STDERR: [Mon Aug 10 03:38:48 2015] [warn] module php5_module is already loaded, skipping
==> vccw.dev:     [Mon Aug 10 03:38:48 2015] [warn] module ssl_module is already loaded, skipping
==> vccw.dev:     Syntax error on line 2 of /etc/httpd/conf.d/README:
==> vccw.dev:     Invalid command 'This', perhaps misspelled or defined by a module not included in the server configuration
==> vccw.dev:     ---- End output of /sbin/service httpd restart && sleep 1 ----
==> vccw.dev:     Ran /sbin/service httpd restart && sleep 1 returned 1
==> vccw.dev:
==> vccw.dev:     Resource Declaration:
==> vccw.dev:     ---------------------
==> vccw.dev:     # In /tmp/vagrant-chef/d9bf11d3e64898221fd7238327d4de7d/cookbooks/apache2/recipes/default.rb
==> vccw.dev:
==> vccw.dev:     221: service "apache2" do
==> vccw.dev:     222:   action :start
==> vccw.dev:     223: end
==> vccw.dev:
==> vccw.dev:     Compiled Resource:
==> vccw.dev:     ------------------
==> vccw.dev:     # Declared in /tmp/vagrant-chef/d9bf11d3e64898221fd7238327d4de7d/cookbooks/apache2/recipes/default.rb:221:in `from_file'
==> vccw.dev:
==> vccw.dev:     service("apache2") do
==> vccw.dev:       action [:start]
==> vccw.dev:       supports {:restart=>true, :reload=>true, :status=>true}
==> vccw.dev:       retries 0
==> vccw.dev:       retry_delay 2
==> vccw.dev:       guard_interpreter :default
==> vccw.dev:       service_name "httpd"
==> vccw.dev:       enabled true
==> vccw.dev:       running true
==> vccw.dev:       pattern "apache2"
==> vccw.dev:       restart_command "/sbin/service httpd restart && sleep 1"
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:   reload_command "/sbin/service httpd reload && sleep 1"
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:   cookbook_name :apache2
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:   recipe_name "default"
==> vccw.dev:
==> vccw.dev:
==> vccw.dev: end
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev:
==> vccw.dev: Running handlers:
==> vccw.dev: [2015-08-10T03:38:48+00:00] ERROR: Running exception handlers
==> vccw.dev: Running handlers complete
==> vccw.dev:
==> vccw.dev: [2015-08-10T03:38:48+00:00] ERROR: Exception handlers complete
==> vccw.dev: [2015-08-10T03:38:48+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
==> vccw.dev: Chef Client failed. 76 resources updated in 1135.384299982 seconds
==> vccw.dev: [2015-08-10T03:38:49+00:00] ERROR: service[apache2] (apache2::default line 221) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
==> vccw.dev: ---- Begin output of /sbin/service httpd restart && sleep 1 ----
==> vccw.dev: STDOUT: Stopping httpd: [FAILED]
==> vccw.dev: Starting httpd: [FAILED]
==> vccw.dev: STDERR: [Mon Aug 10 03:38:48 2015] [warn] module php5_module is already loaded, skipping
==> vccw.dev: [Mon Aug 10 03:38:48 2015] [warn] module ssl_module is already loaded, skipping
==> vccw.dev: Syntax error on line 2 of /etc/httpd/conf.d/README:
==> vccw.dev: Invalid command 'This', perhaps misspelled or defined by a module not included in the server configuration
==> vccw.dev: ---- End output of /sbin/service httpd restart && sleep 1 ----
==> vccw.dev: Ran /sbin/service httpd restart && sleep 1 returned 1
==> vccw.dev: [2015-08-10T03:38:49+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
Chef never successfully completed! Any errors should be visible in the
output above. Please fix your recipes so that they properly complete.
```

Host: OS X 10.10.3
VirtualBox: 4.3.30r101610
Vagrant: 1.7.4
VCCW:  2.13.0

**Comment by tkc49** [comment]
I am also the same problem.
But restart my PC. I was resolved.

**Comment by kakeyang** [comment]
Hi, nice to meet you. Thank you for your activity to develop VCCW.

I also got perfectly same error but the trigger was not changing version of PHP but it was just power outage... I mean when I `vagrant up` and while my laptop was processing it, electricity (and internet) was cut. After that, virtual environment can be updated but apache can't start.
If you could give me any advices, they would be very very helpful for me.
Thanks in advance.

----------------  
Conditions  
----------------  
VCCW: Sorry, I don't know how I can confirm it. But anyway not the latest one (maybe around 2.9).
Mac OS X: 10.9.5
Vagrant: 1.7.4
VirtualBox: 5.0.0r101573  

Here's the result of `vagrant up`.
```
$ vagrant up
Bringing machine 'hostname' up with 'virtualbox' provider...
==> hostname: Checking if box 'miya0001/vccw' is up to date...
==> hostname: VirtualBox VM is already running.
==> hostname: Checking for host entries
```

But after `vagrant ssh`, I can find apache can't start like this.
```
$ service httpd start
Starting httpd: [Fri Aug 21 10:16:18 2015] [warn] module php5_module is already loaded, skipping
[Fri Aug 21 10:16:18 2015] [warn] module ssl_module is already loaded, skipping
Syntax error on line 2 of /etc/httpd/conf.d/README:
Invalid command 'This', perhaps misspelled or defined by a module not included in the server configuration
                                                           [FAILED]
```

I tried to remove `/etc/httpd/conf.d/README` but then I got conflict of port.
```
$ service httpd start
Starting httpd: [Fri Aug 21 10:20:45 2015] [warn] module php5_module is already loaded, skipping
[Fri Aug 21 10:20:45 2015] [warn] module ssl_module is already loaded, skipping
(13)Permission denied: make_sock: could not bind to address [::]:80
(13)Permission denied: make_sock: could not bind to address 0.0.0.0:80
no listening sockets available, shutting down
Unable to open logs
                                                           [FAILED]
```

I didn't understand why so I tried vagrant halt/destroy/up again.
Then I got this error again.
```
================================================================================
Error executing action `restart` on resource 'service[apache2]'
================================================================================
    
Mixlib::ShellOut::ShellCommandFailed
    
------------------------------------
    
Expected process to exit with [0], but received '1'
---- Begin output of /sbin/service httpd restart && sleep 1 ----
STDOUT: Stopping httpd: [FAILED]
Starting httpd: [FAILED]
STDERR: [Fri Aug 21 09:36:53 2015] [warn] module php5_module is already loaded, skipping
[Fri Aug 21 09:36:53 2015] [warn] module ssl_module is already loaded, skipping
Syntax error on line 2 of /etc/httpd/conf.d/README:
Invalid command 'This', perhaps misspelled or defined by a module not included in the server configuration
---- End output of /sbin/service httpd restart && sleep 1 ----
Ran /sbin/service httpd restart && sleep 1 returned 1
    
    Resource Declaration:
    ---------------------
    # In /tmp/vagrant-chef/96c6cde3002592395adee7e8f38a618d/cookbooks/apache2/recipes/default.rb
    
    221: service "apache2" do
    222:   action :start
    223: end
    
    Compiled Resource:
    ------------------
    # Declared in /tmp/vagrant-chef/96c6cde3002592395adee7e8f38a618d/cookbooks/apache2/recipes/default.rb:221:in `from_file'
    
    service("apache2") do
      action [:start]
      supports {:restart=>true, :reload=>true, :status=>true}
      retries 0
      retry_delay 2
      guard_interpreter :default
      service_name "httpd"
      enabled true
      running true
      pattern "apache2"
      restart_command "/sbin/service httpd restart && sleep 1"
      reload_command "/sbin/service httpd reload && sleep 1"
      cookbook_name :apache2
      recipe_name "default"
    end

Running handlers:
[2015-08-21T09:36:53+00:00] ERROR: Running exception handlers
Running handlers complete

[2015-08-21T09:36:53+00:00] ERROR: Exception handlers complete
[2015-08-21T09:36:53+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
Chef Client failed. 64 resources updated in 1606.329441304 seconds
[2015-08-21T09:36:53+00:00] ERROR: service[apache2] (apache2::default line 221) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of /sbin/service httpd restart && sleep 1 ----
STDOUT: Stopping httpd: [FAILED]
Starting httpd: [FAILED]
STDERR: [Fri Aug 21 09:36:53 2015] [warn] module php5_module is already loaded, skipping
[Fri Aug 21 09:36:53 2015] [warn] module ssl_module is already loaded, skipping
Syntax error on line 2 of /etc/httpd/conf.d/README:
Invalid command 'This', perhaps misspelled or defined by a module not included in the server configuration
---- End output of /sbin/service httpd restart && sleep 1 ----
Ran /sbin/service httpd restart && sleep 1 returned 1
[2015-08-21T09:36:53+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)

Chef never successfully completed! Any errors should be visible in the
output above. Please fix your recipes so that they properly complete.
```

**Comment by miya0001** [comment]
Please try,

```
$ vagrant reload
$ vagrant provision
```

**Comment by kakeyang** [comment]
Oh apache runs! Thank you very much!

**Comment by miya0001** [comment]
Hi,

I found a cause of this problem, so I've released v2.14.0.
Thanks guys!

https://github.com/vccw-team/vccw/releases/tag/2.14.0

**Comment by katanyan** [comment]
Great thanks for @tkc49 and @miya001. It works fine for me.
